css/home.css

```diff
@@ -98,6 +98,10 @@ body {
   font-size: 12px;
   font-weight: regular;
 }
+.container {
```

Comment:
Hey @jhonatanrojasbastidas I think you must set this class into the style.css file because this is going to be used in all pages, no just home.css

Reply:
Agree this one can be an excellent utility class, nice one.

Reply:
after reviewing the use of this class with @christiang6 i would suggest to add these rules to it, `height: 100%;`and `position: relative;` so it works right from the start with our general section structure.

what do you think @jhonatanrojasbastidas , @ArleyAV , @josemariogutierrez ?

Reply:
@jusk02  that rules is for the work in general, i answer because i dont look any change when i applied in the structure of the carrousel div¿?

Reply:
Not sure if `height: 100%`

Reply:
Arley already included the `.container` class inside `style.css` and is now merged in develop, so is not needed in `home.css`, you can delete it.

Reply:
I think the container should always scale to the height of its parent div/section, and if it must have a fixed different height it should be set within its component specific classes.

But not setting `height: 100%;` and setting the height manually in each case also works.